src/main/java/ay2021s1_cs2103_w16_3/finesse/storage/JsonSerializableFinanceTracker.java

```diff
@@ -32,10 +34,13 @@
     public JsonSerializableFinanceTracker(@JsonProperty("expenses") List<JsonAdaptedExpense> expenses,
                                           @JsonProperty("incomes") List<JsonAdaptedIncome> incomes,
                                           @JsonProperty("frequentExpenses") List<JsonAdaptedFrequentExpense>
```

Comment:
```suggestion
            @JsonProperty("incomes") List<JsonAdaptedIncome> incomes,
            @JsonProperty("frequentExpenses") List<JsonAdaptedFrequentExpense>
```
Follow this indentation for the lines below as well.